test/functions/helloworld.test.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var assert = require('assert');
+
+var helloworldSample = require('../../functions/helloworld');
+
+describe('functions/helloworld', function () {
```

Comment:
Sigh. More code to test than to write. I generally abhor these kinds of tests as its usually more of a burden to maintain the tests than the code, but it's fine. I'm just venting.